buildkite/src/Constants/ContainerImages.dhall

```diff
@@ -4,11 +4,11 @@
 -- NOTE: minaToolchainBookworm is also used for building Ubuntu Jammy packages in CI
 { toolchainBase = "codaprotocol/ci-toolchain-base:v3"
 , minaToolchainBullseye =
-    "gcr.io/o1labs-192920/mina-toolchain@sha256:e537f1fb060ddb5b5db2270a4220f3d777c31c175a0480a5b0c4dc66433c0726"
+    "gcr.io/o1labs-192920/mina-toolchain@sha256:88005de885408b94a14ddec7614a7e1389c926e29909c503294e2cd85aab0ae8"
```

Comment:
I assume these are prebuilt artifacts -- is this updated automatically or manually? in any case, is it possible to make any sense of these? (they don't have any version number, just she hash, so it's confusing to me...)

Reply:
The artefacts are built using `!ci-toolchain-me`, see [this comment](https://github.com/MinaProtocol/mina/pull/16652#issuecomment-2673968292). The builds are after that available in buildkite [here](https://buildkite.com/o-1-labs-2/mina-toolchains-build/builds/127). The hash comes from the job "Docker: toolchain-focal-docker-image".